### ISSUE TITLE
Add destinationPredefinedAcl to copy and move

### ISSUE THREAD

**Issue by jeremejevs** [GoogleCloudPlatform/google-cloud-node#1937]
Would be nice to be able to do something like:

```js
await tmpFile.copy(dstFile, { destinationPredefinedAcl: 'publicRead' })
```

Instead of:

```js
await tmpFile.copy(dstFile)
await dstFile.acl.readers.addAllUsers()
```

This isn't the only option missing, of course, but I'm just reporting my usecase.

**Comment by stephenplusplus** [comment]
We should definitely do that! PR welcome :)

https://github.com/GoogleCloudPlatform/google-cloud-node/blob/4d4163fa59a4452b9762482b032921994edda17f/packages/storage/src/file.js#L464
```diff
- var query = {};
+ var query = extend({}, options);
if (is.defined(this.generation)) {
  query.sourceGeneration = this.generation;
}
if (is.defined(options.token)) {
  query.rewriteToken = options.token;
+ delete query.token;
}
```

**Comment by stephenplusplus** [comment]
This will be fixed as part of #2001.

**Comment by jeremejevs** [comment]
Thanks!
